treeherder/push_health/utils.py

```diff
@@ -8,15 +8,9 @@
 ]
 
 
-def clean_test(action, test, signature, message):
+def clean_test(test, signature, message):
     try:
-        clean_name = 'Non-Test Error'
-        if action == 'test_result':
-            clean_name = test
-        elif action == 'crash':
-            clean_name = signature
-        elif action == 'log':
-            clean_name = message
+        clean_name = test or signature or message or 'Non-Test Error'
```

Comment:
This is the primary fix.  Instead of just always using the `signature` when it's a crash, we now use the `test` if it exists.  But fall back to the other items, if that's all we have.

Reply:
So this is now going to group crashes under a specific test path instead of dumping it out by crash message in its own section? Is a signature the crash message (for ex: "Shutdown Leaks | process called before end of suite" per Anny's screenshot)?

Reply:
It will still put crashes into the `crashes` section.  Won't mix it in with the Tests.  We *could* do that, but I'm not really sure if that's preferable.  I wasn't sure from Anny's bug comments.